src/test/java/org/minions/devfund/danielmontecinos/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,168 @@
+package org.minions.devfund.danielmontecinos.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Tests MovieDatabase class.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase movieDatabase;
+
+    /**
+     * Creates a MovieDatabase object.
+     */
+    @Before
+    public void setUp() {
+        movieDatabase = new MovieDatabase();
+    }
+
+    /**
+     * Tests movieList is empty when no movie is added.
+     */
+    @Test
+    public void testGetMovieListIsEmptyWhenNoMoviesAreAdded() {
+        assertTrue(movieDatabase.getMovieList().isEmpty());
+    }
+
+    /**
+     * Tests actorList is empty when no movie is added.
+     */
+    @Test
+    public void testGetActorListIsEmptyWhenNoMoviesAreAdded() {
+        assertTrue(movieDatabase.getActorList().isEmpty());
+    }
+
+    /**
+     * Tests addMovie method add movie names and movie actors.
+     */
+    @Test
+    public void testAddMovieMethodAddsANewMovieWithAListOfActors() {
+        movieDatabase.addMovie("Movie 1", new String[]{"Actor 1", "Actor 2", "Actor 3"});
+        final int expectedSize = 1;
+
+        assertEquals("Movie 1", movieDatabase.getMovieList().get(0).getName());
+        assertEquals(expectedSize, movieDatabase.getMovieList().size());
+
+        assertEquals("Actor 1", movieDatabase.getMovieList().get(0).getActors().get(0).getName());
+        assertEquals("Actor 2", movieDatabase.getMovieList().get(0).getActors().get(1).getName());
+        assertEquals("Actor 3", movieDatabase.getMovieList().get(0).getActors().get(2).getName());
+    }
+
+    /**
+     * Tests that addMovie method doesn't add an exiting movie to movieList.
+     */
+    @Test
+    public void testAddMovieMethodDoesNotAddSameMovies() {
+        movieDatabase.addMovie("Movie 1", new String[]{"Actor 1", "Actor 2", "Actor 3"});
+        movieDatabase.addMovie("Movie 1", new String[]{"Actor 4", "Actor 5", "Actor 6"});
+
+        final int expectedSize = 1;
+        assertEquals(expectedSize, movieDatabase.getMovieList().size());
+    }
+
+    /**
+     * Tests that addMovie method can add different movies to movieList.
+     */
+    @Test
+    public void testAddMovieMethodAddsDifferentMovies() {
+        movieDatabase.addMovie("Movie 4", new String[]{"Actor 7"});
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Actor 7" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)